Author: Adam Di Carlo <aph@debian.org>
Description: The removal of ArtHeader in 4.0 is not documented in ChangeLog.
 This patch fixes it.
Bug-Debian: http://bugs.debian.org/bug=101779

--- a/ChangeLog.upstream
+++ b/ChangeLog.upstream
@@ -475,6 +475,7 @@
 - Added synopsis markup for modern programming languages (e.g, object
   oriented languages like Java, C++, and IDL)
 - Renamed DocInfo to PrefaceInfo, ChapterInfo, AppendixInfo, etc.
+  (and ArtHeader to ArticleInfo)
 
 Other:
 
--- a/docbook-4.1/40chg.txt
+++ b/docbook-4.1/40chg.txt
@@ -29,6 +29,7 @@
 - Added synopsis markup for modern programming languages (e.g, object
   oriented languages like Java, C++, and IDL)
 - Renamed DocInfo to PrefaceInfo, ChapterInfo, AppendixInfo, etc.
+  (and ArtHeader to ArticleInfo)
 - Comment was renamed Remark
 - InterfaceDefinition was removed
 
